src/Umbraco.Web.UI.Client/src/common/directives/components/umbdatetimepicker.directive.js

```diff
@@ -0,0 +1,143 @@
+/**
+@ngdoc directive
```

Comment:
Perhaps with the doc comment, indicate what version it will be for. In case package devs try to use it in an older version of Umbraco.